src/page/Variant.tsx

```diff
@@ -185,7 +186,7 @@ class Variant extends React.Component<IVariantProps> {
                         [TrackName.Exon]: 'visible',
                         [TrackName.UniprotTopology]: 'visible',
                     }}
-                    hugoSymbol={mutation[0].gene.hugoGeneSymbol}
+                    hugoSymbol={mutation[0]?.gene?.hugoGeneSymbol}
```

Comment:
Not harmful, but we already have undefined checks above. https://github.com/genome-nexus/genome-nexus-frontend/blob/master/src/page/Variant.tsx#L154-L157